src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlEventSourceListener.netfx.cs

```diff
@@ -153,6 +153,15 @@ private void OnBeginExecute(EventWrittenEventArgs eventData)
                     activity.SetTag(SemanticConventions.AttributeDbStatement, commandText);
                 }
             }
+            
+            try
+            {
+                this.options.Enrich?.Invoke(activity, "OnCustom", eventData.Payload);
```

Comment:
what is the use case for this payload? the instrumentaion already adds everything useful from it as tags - are we missing anything?

Reply:
Nothing I am aware of, I cannot think of any reason to use the SqlCommand either, but it encapsulates the same idea as the SqlCommand that is passed from .NET in providing the raw data from with the activity is created from.

Reply:
just saw similar comment was already made: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4161/files#r1098995182